services/context/access_log.go

```diff
@@ -86,6 +86,7 @@ func AccessLogger() func(http.Handler) http.Handler {
 				ResponseWriter: rw,
 				Ctx: map[string]any{
 					"RemoteAddr": req.RemoteAddr,
+					"UserAgent":  req.UserAgent,
```

Comment:
Are you sure it is right? IIRC `req.UserAgent` is a function.

Reply:
This is code that came in from a different branch I'm working on. I've removed it.